content/blog/improved-preview-experience/index.md

```diff
@@ -0,0 +1,218 @@
+---
+title: "Improved Pulumi Previews"
+date: 2020-10-29
+draft: false
+meta_desc: "Announcing a significant improvement to the pulumi preview experience."
+meta_image: preview_update.png 
+authors:
+    - paul-stack
+tags:
+    - enhancement
+    - pulumi
+---
+
+Today we are announcing a minor but significant improvement to the Pulumi [preview]({{< relref "/docs/reference/cli/pulumi_preview" >}})
```

Comment:
This is a bit of an oxymoron. Perhaps rephrased to:

"Today we are announcing an improvement to the Pulumi preview experience which provides a significant increase in the user experience"